### PR TITLE
feat: drop organizations

### PR DESCRIPTION
**Pull request by ivan-tymoshenko**

### CODE REVIEW

File: __mocks__/templates.json [hunk 1]
```diff
@@ -26,47 +26,5 @@
     "author": "Platformatic",
     "homepage": "https://example.com",
     "public": true
```
Comment (marcopiraccini) [inline __mocks__/templates.json:28]: Why do we still have this mock @ivan-tymoshenko @tonysnowboardunderthebridge ?

**Comment by marcopiraccini** [comment]
@ivan-tymoshenko I don't understand, why are we removing the API key here for the calls? If we do that we cannot see the private stackables anymore, right?

**Comment by ivan-tymoshenko** [comment]
yes, we are dropping private stackables

File: __mocks__/templates.json (hunk 1 above)
Comment (ivan-tymoshenko) [reply __mocks__/templates.json:28]: Do you mean the "public" property or the mock whole mock file?

**Review by mcollina — APPROVED** [review]
lgtm

**Review by marcopiraccini — APPROVED** [review]
lgtm

**Pull request merged**